torch/distributed/_functional_collectives.py

```diff
@@ -15,6 +15,17 @@
     get_innermost_proxy_mode,
 )
 
+"""
+The old check `sys.executable == 'torch_deploy'` for torch::deploy was replaced here:
+https://github.com/pytorch/multipy/pull/138/files#diff-dae4e20139ff6af007a16cc6888d0e3c1d40d297cb7aef89d8e6cc201caacb9eR124
+
+The new check is a bit more hackish, but that's all we have for now.
+
+"""
+def _is_running_under_torch_deploy():
+    return torch._meta_registrations is object
```

Comment:
Could you send a PR to multipy to add a comment next to that line saying that we rely on this particular "object" state and it should not be modified?

Reply:
https://github.com/kumpera/multipy/pull/1

Reply:
https://github.com/pytorch/multipy/pull/317